### DISCUSSION TITLE
Parablade - TurboDesigner collab

### DISCUSSION THREAD

**Discussion by afshawnlotfi** [NAnand-TUD/parablade#20] [Show and tell]
Hi guys,

I have been watching your excellent project Parablade for a while and was working on my own similar project TurboDesigner (https://github.com/Turbodesigner/turbodesigner) which takes in axial turbomachinery specifications and generates the CAD for it. Recently Turbodesigner has been gaining some steam and I was hoping if we could collaborate together on making an open source turbomachinery tool that does the whole pipeline from specification, to meanline analysis, to flow and CFD analysis (similar to Parablade), to optimization via multi-objective or reinforcement learning, and to finally generating the CAD assembly with a Bill of Materials. Please let me know if this is something you'd be interested in.

Warm Regards,
Afshawn Lotfi

**Comment by NAnand-TUD** [comment]
Hi Afshawn,

Thanks for your message. Roberto and I are busy with other projects currently, however, we have coupled parablade with mean-line solvers, but that was never made public. However, if you wish to extend please feel free and if you need any interaction points (which are small efforts) I can help. Perhaps more importantly, I want parablade to be free from commercialising or using any commercial software. 

Thanks,
Nitish
